lib/util/tXml.js

```diff
@@ -0,0 +1,707 @@
+goog.provide('shaka.util.TXml');
+
+goog.require('shaka.util.StringUtils');
+goog.require('shaka.log');
+
+/**
+ * This code is a modified version of the tXml library.
+ *
+ * @author: Tobias Nickel
+ * created: 06.04.2015
+ * https://github.com/TobiasNickel/tXml
+ */
+
+shaka.util.TXml = class {
+  /**
+     * Parse some data
+     * @param {BufferSource} data
+     * @param {string=} expectedRootElemName
+     * @return {shaka.extern.xml.Node | null}
+     */
+  static parseXml(data, expectedRootElemName) {
+    const xmlString = shaka.util.StringUtils.fromBytesAutoDetect(data);
+    return shaka.util.TXml.parseXmlString(xmlString, expectedRootElemName);
+  }
+
+  /**
+   * Parse some data
+   * @param {string} xmlString
+   * @param {string=} expectedRootElemName
+   * @return {shaka.extern.xml.Node | null}
+   */
+  static parseXmlString(xmlString, expectedRootElemName) {
```

Comment:
conformance.textproto mentions shaka.util.XmlUtils.parseXmlString directly. You should update that to mention shaka.util.TXml.parseXmlString instead.